### ISSUE TITLE
Training hyper parameters and pre-trained model

### ISSUE THREAD

**Issue by Ryougetsu3606** [seorim0/SE-using-SRL-Model#4]
Hello! 

Thanks for your excellent work! I encountered some difficulty when replicate the Stage2 model, where the highest PESQ score is 0.2 lower than proposed in the article. I believe there is something wrong in my setting (like learning rate and batch size), so could you please share the pretrained weight or training details?

P.S. I use a win_len of 512ms when replicating, which is the same as the setting in your code, but different from the one in your paper. I am not sure if it will make great difference in the result because it seems not influence the Stage1 training.

**Comment by 1zxLi** [comment]
Thanks for sharing your results. If you still have the trained checkpoint, would you be willing to share it? I’d really appreciate the chance to test it and compare with my setup.


Thanks!
